dev-tools/cherrypick_pr

```diff
@@ -145,8 +152,16 @@ def main():
         new_pr = request.json()
 
         # add labels
+        labels = ["backport"]
+
+        if args.zube_team:
+            labels.append("Team:"+args.zube_team)
```

Comment:
What happens if a label doesn't exist? Does it create it or it fails? I think it should fail in this case. If not I am afraid that typos in the team can end-up creating lots of team labels.

Reply:
Good point. I've added a check.

Reply:
Thanks!